src/main/webapp/app/entities/programming-exercise/programming-exercise-detail.component.html

```diff
@@ -68,11 +68,27 @@ <h2><span jhiTranslate="arTeMiSApp.programmingExercise.detail.title">Programming
                 </dd>
                 <dt><span jhiTranslate="arTeMiSApp.programmingExercise.templateResult">Template Result</span></dt>
                 <dd>
-                    <div *ngIf="programmingExercise.templateParticipation.results"><jhi-result [participation]="programmingExercise.templateParticipation"></jhi-result></div>
+                    <div *ngIf="programmingExercise.templateParticipation.results" class="d-flex align-items-center">
+                        <jhi-result [participation]="programmingExercise.templateParticipation" class="mr-2"></jhi-result>
+                        <jhi-programming-exercise-instructor-status
+                            *ngIf="programmingExercise.templateParticipation.results.length"
```

Comment:
Is this already working with the new case, that we store positive feedback as well? I guess you need to filter for negative results here, right?